lib/overcommit/subprocess.rb

```diff
@@ -85,14 +82,28 @@ def win32_prepare_args(args)
 
       # @param process [ChildProcess]
       # @return [Array<IO>]
-      def assign_output_streams(process)
+      def assign_output_streams(process, options = {})
+        if options[:live_output]
+          process.io.inherit!
+          return []
+        end
+
         %w[out err].map do |stream_name|
           ::Tempfile.new(stream_name).tap do |stream|
             stream.sync = true
             process.io.send("std#{stream_name}=", stream)
           end
         end
       end
+
+      def build_result(process, out, err, options = {})
+        return Result.new(process.exit_code, '', '') if options[:live_output]
```

Comment:
This can potentially break any processing done by the hook. If the `exit_code` is non-zero, hooks can sometimes read the stderr/stdout to determine the kind of failure that occurred. Given all the different ways Overcommit hooks _could_ be used, we can't introduce a change that fundamentally breaks the behavior.